src/cli/parse-flags.ts

```diff
@@ -132,6 +132,7 @@ const ARG_OPTS = {
     'log',
     'open',
     'prerender',
+    'service-worker',
```

Comment:
Would you be able to add unit tests for the service worker flag? https://github.com/ionic-team/stencil/blob/master/src/cli/test/parse-flags.spec.ts